Add register_command and register_steam docs (#39)
Drachenfrucht1/godotcord: GodotcordActivityManager.md

`clear_activity()`
- clears the users activity

`register_command(String command)`

> - used to register command to run game by which Discord can launch your game. Recommended to check [discord](https://discord.com/developers/docs/game-sdk/activities#registercommand)'s docs.

Reviewer: ```suggestion
- Used to register a command that will allow Discord to launch your game when needed (for example when someone wants to join a person's game)
- The command can either be a protocol (`my-godot-game://`) or a path to your game's executable.
- The command also supports launch parameters: `path/to/my/game/godotgame.exe --fullscreen --fixed-fps 60`
```

`clear_activity()`
- clears the users activity

`register_command(String command)`
- Used to register a command that will allow Discord to launch your game when needed (for example when someone wants to join a person's game)
- The command can either be a protocol (`my-godot-game://`) or a path to your game's executable.
- The command also supports launch parameters: `path/to/my/game/godotgame.exe --fullscreen --fixed-fps 60`
- More info on the [Game SDK docs](https://discord.com/developers/docs/game-sdk/activities#registercommand)

`register_steam(int steam_id)`

> - used to register game that has been distributed using Steam. Check [discord](https://discord.com/developers/docs/game-sdk/activities#registersteam)'s docs for more info. 

Reviewer: ```suggestion
- Used to register a Steam game to be launched when needed. Check [discord](https://discord.com/developers/docs/game-sdk/activities#registersteam)'s docs for more info. 
```
